packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```
@@ -426,6 +432,9 @@ Resources:
           PolicyName: default
       RoleName:
         Fn::Sub: cdk-${Qualifier}-deploy-role-${AWS::AccountId}-${AWS::Region}
+      Tags:
+        - Key: aws-cdk:bootstrap-role
+          Value: deploy-artifacts
```

This is a bit of a misnomer. This role doesn't just deploy artifacts, it deploys everything. Maybe just
```suggestion
          Value: deploy
```

To match the naming schema of the other roles?